tapir/shifts/templates/shifts/shift_detail.html

```diff
@@ -163,15 +163,17 @@ <h5 class="d-flex justify-content-between align-items-center">
                                                 qualification, contact the member office)<br/>
                                                 -The shift is in the future<br/>
                                             {% endblocktranslate %}
-                                            <span {% if not slot.can_register %} data-bs-toggle="tooltip" data-html="true"
+                                        {% autoescape off %}
```

Comment:
Otherwise, Bootstrap will render python escape chars and not the HTML content.

Reply:
Ah, now I get it.

Seems like a garbage design decision by Bootstrap, turning off autoescaping is potentially a dangerous move in every framework... but obviously fine in this case since we're not dealing with user input :)